<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manage</class>
 <widget class="QDialog" name="manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel管理程序</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size:18px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,8">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请选择Excel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_excel">
       <property name="text">
        <string>excel文件未选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>学号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_select">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>学号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>语文</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数学</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>英语</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_single_data">
     <property name="text">
      <string>新增学生数据</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="remove_data">
     <property name="text">
      <string>删除学生数据</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="change_data">
     <property name="text">
      <string>修改学生数据</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_show">
     <property name="text">
      <string>展示统计数据</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
